Modules/_math.h

```diff
@@ -23,3 +23,21 @@ _Py_log1p(double x)
 }
 
 #define m_log1p _Py_log1p
+
+static double
+_Py_fmod(double x, double y)
```

Comment:
There is a "‘_Py_fmod’ defined but not used [-Wunused-function]" warning. Maybe because _math.h is included by Modules/cmathmodule.c which doesn't use it?

Maybe move this function to mathmodule.c?

Reply:
> IIUC, this is the place where various workaround for buggy libm kept.

I understand, but you should find a way to avoid the warning. Maybe don't declare the function if it's included by cmathmodule.c?

Reply:
Perhaps, it's fine to inline this to the mathmodule.c, as it's not used outside.

Or I could add ``_math.h`` to ``Tools/build/.warningignore*``.

What better? :)

Reply:
> Or I could add _math.h to Tools/build/.warningignore*.

Please don't do that. These files are only used by CIs, but developers don't use it. A new warning would annoy everyone.